src/test/cls_sdk/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+add_executable(ceph_test_cls_sdk
+  test_cls_sdk.cc
+  )
+set_target_properties(ceph_test_cls_sdk PROPERTIES COMPILE_FLAGS
+  ${UNITTEST_CXX_FLAGS})
+target_link_libraries(ceph_test_cls_sdk
+  librados
+  global
+  ${EXTRALIBS}
+  ${BLKID_LIBRARIES}
+  ${CMAKE_DL_LIBS}
+  radostest
+  ${UNITTEST_LIBS}
+  )
+install(TARGETS
```

Comment:
@neha-ojha you cannot do both. if you choose to run this test as part of "make check", then "add_ceph_unittest()", and there is no necessary to run it in `qa/workunits/cls/test_cls_sdk.sh`,  but if you choose otherwise, you need to package this test by `install(TARGET ...)`, so `qa/workunits/cls/test_cls_sdk.sh` can exercise it in a qa suite.